scripts/test-operator-integration.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+# Check if the multi-operator-manager is installed; if not, install it
+if ! command -v multi-operator-manager &> /dev/null; then
```

Comment:
How can I make this command download a new MOM?  If we don't do a `go install`, output the command I need to run in order to download on the next invocation